client/src/components/form/DragAndDrop.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { RiDeleteBinLine, RiFileLine, RiInformation2Line } from '@remixicon/react';
+import { Callout, Divider, Select, SelectItem, TextInput } from '@tremor/react';
+import { useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+interface File {
+    path: string;
+    size: number;
+}
+
+function classNames(...classes: string[]) {
+    return classes.filter(Boolean).join(' ');
+}
+
+export function FileUpload() {
+    const [showAttachmentInfo, setShowAttachmentInfo] = useState(false);
```

Comment:
I selected a file about 1.4 Gbytes and there were no control for that to forbid uploading. ( Max. size per file: 50MB ) 
<img width="1440" alt="Screen Shot 2024-11-12 at 18 30 41" src="https://github.com/user-attachments/assets/e243a72a-854b-4b51-b108-cf42c23c4936">

<img width="1440" alt="Screen Shot 2024-11-12 at 18 26 17" src="https://github.com/user-attachments/assets/f7a3f2ec-ee53-49a1-bd44-ba333e638f55">

Reply:
yes, because it's only the UI part, there is another task for everythig else